.github/actions/setup-jimm/action.yml

```diff
@@ -0,0 +1,62 @@
+name: Set up JIMM
+description: Set up JIMM with provided configuration.
+inputs:
+  jimm-checkout-location:
+    description: Path to checkout the JIMM repository to.
+    required: false
+    default: jimm
+  jimm-version:
+    description: Version of JIMM to use.
+    required: false
+    default: dev
+  juju-channel:
+    description: Channel of Juju to use.
+    required: false
+    default: 3.6/edge
```

Comment:
If this workflow passes with "3/stable" maybe that would be a better default (the k8s version has trouble with 3.6/edge).